templates/zerver/create_realm.html

```diff
@@ -4,10 +4,29 @@
 {# This is where we pitch the app and solicit signups. #}
 
 {% block portico_content %}
+<script src='https://www.google.com/recaptcha/api.js'></script>
```

Comment:
This doesn't follow the examples in the documentation, and it won't work reliably. The fancy deferred ways of using the API have specific requirements on the way this `<script>` element is included. See https://developers.google.com/recaptcha/docs/invisible , and search for "race condition".

Reply:
Also we should leave this element out when reCAPTCHA is not configured.